tests/rptest/tests/raft_availability_test.py

```diff
@@ -362,6 +362,42 @@ def test_leader_restart(self):
             time.sleep(ELECTION_TIMEOUT)
             assert new_leader_id == self._get_leader()[0]
 
+    @cluster(num_nodes=3)
+    def test_leadership_transfer(self):
```

Comment:
nit: we have leadership_transfer_test.py, maybe better move this test there?

Reply:
here i have all necessary util functions :)